### PR TITLE
HW_4_Object_concept

### PR DESCRIPTION
**Pull request by SerjPyth**
Concise code in gathered in file "homework". Since there were no tests, I`m also including files for every class separately, which contain my testing process and different variations of solutions. I found two weak spots, which I was not able to remove by this moment:
1) Cesar_class can`t handle a scenario with only 1 Garage in ownership. 
2) Method Cesar.add_car_ces(self, garage, car) doesn`t work when garage = None.

### CODE REVIEW

File: objects_and_classes/homework/homework.py [hunk 1]
```diff
@@ -47,16 +47,230 @@
...
-class Cesar:
+
+class WrongException(Exception):
     pass
 
 
 class Car:
-    pass
+
+    def __init__(self, price, type, producer, number, mileage):
```
Comment (Zaplatynskyi) [inline objects_and_classes/homework/homework.py]: value is not used

File: objects_and_classes/homework/homework.py [hunk 2]
```diff
@@ -47,16 +47,230 @@
...
+            for car in self.cars:
+                single = car.get_price()
+                total.append(single)
+                i -= 1
+            return sum(total)
+
+
+class Cesar:
+
+    def __init__(self, name, register_id, garages=None):
```
Comment (Zaplatynskyi) [inline objects_and_classes/homework/homework.py]: value in not used

File: objects_and_classes/homework/homework.py [hunk 3]
```diff
@@ -47,16 +47,230 @@
...
+                single = garage.hit_hat()
+                total.append(single)
+                i -= 1
+            return sum(total)
+
+    def garage_count(self):
+        i = 0
+        j = len(self.garages)
+        while j != 0:
+            for garage in self.garages:
```
Comment (Zaplatynskyi) [inline objects_and_classes/homework/homework.py]: looks so strange:) 
it is possible to do without a loop, using the length of the list

File: objects_and_classes/homework/homework.py (hunk 1 above)
Comment (SerjPyth) [reply objects_and_classes/homework/homework.py]: Removed number value

File: objects_and_classes/homework/homework.py (hunk 2 above)
Comment (SerjPyth) [reply objects_and_classes/homework/homework.py]: Removed register_id value

File: objects_and_classes/homework/homework.py (hunk 3 above)
Comment (SerjPyth) [reply objects_and_classes/homework/homework.py]: Changed to boring "return len(self.garages)" )

**Review by Zaplatynskyi — APPROVED** [review]
looks good for me:)

**Pull request merged**